### ISSUE TITLE
Trouble with starting the plugin the world is read as null ?

### ISSUE THREAD

**Issue by WojciechKosierkiewicz** [BlueNightFury46/Hardcore24#5]
I'm not sure if I configured it correctly or if something is wrong with parsing the config, can you help me please ?
```
[17:13:10] [Server thread/INFO]: [hardcore24] Enabling Hardcore24 v1.05c
[17:13:10] [Server thread/INFO]: [hardcore24] FATAL ERROR! RESPAWN WORLD IS NULL
[17:13:10] [Server thread/INFO]: [hardcore24] Disabling Hardcore24!
[17:13:10] [Server thread/INFO]: [hardcore24] Starting Hardcore24 by BlueNightFury46
[17:13:10] [Server thread/INFO]: [hardcore24] Hardcore24 is active and should be working as usual
[17:13:10] [Server thread/INFO]: [hardcore24] Disabling Hardcore24!
[17:13:10] [Server thread/INFO]: [hardcore24] FATAL ERROR! THE RESPAWN WORLD IN THE CONFIG DOES NOT EXIST! TRY CHANGING IT AND RESTARTING THE SERVER!

 NOTE: IT MAY BE CASE SENSITIVE
[17:13:10] [Server thread/ERROR]: Error occurred while enabling Hardcore24 v1.05c (Is it up to date?)
java.lang.NullPointerException: Cannot invoke "org.bukkit.World.getGameRuleValue(org.bukkit.GameRule)" because "uk.co.shadowtrilogy.hardcore24.Hardcore24.OVERWORLD" is null
        at Hardcore24.jar/uk.co.shadowtrilogy.hardcore24.Hardcore24.onEnable(Hardcore24.java:150) ~[Hardcore24.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:288) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[paper-1.21.1.jar:1.21.1-33-64c9ee6]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.21.1.jar:1.21.1-33-64c9ee6]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:640) ~[paper-1.21.1.jar:1.21.1-33-64c9ee6]
        at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:589) ~[paper-1.21.1.jar:1.21.1-33-64c9ee6]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:293) ~[paper-1.21.1.jar:1.21.1-33-64c9ee6]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1215) ~[paper-1.21.1.jar:1.21.1-33-64c9ee6]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:330) ~[paper-1.21.1.jar:1.21.1-33-64c9ee6]
        at java.base/java.lang.Thread.run(Thread.java:1570) ~[?:?]
[17:13:10] [Server thread/INFO]: [hardcore24] Disabling Hardcore24 v1.05c
[17:13:10] [Server thread/INFO]: [hardcore24] Disabling Hardcore24!
```
```
respawn-location:
  x: 0.0
  y: 0.0
  z: 0.0
  world: 'world'
# The world they get sent to after respawning
#
hardcore-world:
  hardcore-normal: 'world'
  hardcore-nether: 'world_nether'
  hardcore-end: 'world_end'
# Configure the hardcore worlds
# Note: If you don't have a nether and end world then rename them to a world that doesn't exist
hardcore-config:
  server-timezone: 'GMT'
  harder-mobs: false
  do-blood-moon: false
  # ^ Enables the optional Blood-Moon feature. The Blood Moon spawns full Netherite mobs every couple full Moons for a specified time.
  blood-moon-duration: 2.0
  # in hrs (above)
  death-ban-time: 1.0
  # in hours, above ^
  allow-instant-respawn: true
  # Recommended, turns on instant respawn for the hardcore worlds, which skips the respawn screen.
  can't-place-shulker-during-damage: false
  # Warning! The above feature is experimental and may cause issues
  death-ban-exclude-ops: true
  # If true operators will be excluded from the death ban
hardcore-message-config:
  blood-moon-message: '§4As the blood moon rises the countless monsters band together, you encounter mobs with stronger gear...§4'
  blood-moon-title-message: '§cThe Blood Moon Rises Once Again!§c'
  blood-moon-subtitle-message: '§4Mobs you find will have stronger armour§4'
```

**Comment by BlueNightFury46** [comment]
Ah, okay! Sorry, that was a broken build that slipped past me, I'll release
a patch tommorow.

Wojciech ***@***.***> schrieb am So., 18. Aug. 2024, 22:36:

**Comment by BlueNightFury46** [comment]
Okay, it's fixed now! Please download the beta build (1.06a)
